### ISSUE TITLE
Swap out app for Pushover or similar mobile app

### ISSUE THREAD

**Issue by rudikershaw** [rudikershaw/water-alarm-app#1]
https://pushover.net/ does a push notifications as a service, plus provided mobile app type thing. Look into replacing the broken app in this repository with one of these kinds of service.

Once that is done you should be able to finish off the READMEs.

**Comment by rudikershaw** [comment]
Or maybe? 

- pushed
- pushbullet
- pushalot

**Comment by rudikershaw** [comment]
Done and merged into master.
